gensrc/script/doris_builtins_functions.py

```diff
@@ -531,6 +531,8 @@
     [['rpad'], 'VARCHAR', ['VARCHAR', 'INT', 'VARCHAR'],
             '_ZN5doris15StringFunctions4rpadEPN9doris_udf'
             '15FunctionContextERKNS1_9StringValERKNS1_6IntValES6_'],
+    [['append_trailing_char_if_absent'], 'VARCHAR', ['VARCHAR', 'VARCHAR'],
```

Comment:
Could you tell me in what scenario this function is used?

Reply:
may be such like this : 
```
MySQL [test]> select append_trailing_char_if_absent('http://google.com','/');
+----------------------------------------------------------+
| append_trailing_char_if_absent('http://google.com', '/') |
+----------------------------------------------------------+
| http://google.com/                                       |
+----------------------------------------------------------+
1 row in set (0.01 sec)

MySQL [test]> select append_trailing_char_if_absent('http://google.com/','/');
+-----------------------------------------------------------+
| append_trailing_char_if_absent('http://google.com/', '/') |
+-----------------------------------------------------------+
| http://google.com/                                        |
+-----------------------------------------------------------+
```